xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -227,7 +227,7 @@ Reads exactly 2 bytes from the beginning of the span.
       </Parameters>
       <Docs>
         <param name="source">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Reads a <see cref="T:System.Int128" /> from the beginning of a read-only span of bytes, as big endian.</summary>
         <returns>To be added.</returns>
```

Comment:
All these empty `<param>` and `<returns>` items will keep showing the APIs as undocumented. Any chance you can add them?

Reply:
Can I do that in a secondary PR? There's a few bits missing from all of these, so it'd be nice to get what we have merged. Get a new diff of only what's truly missing (across all my areas) and then do a second batch of PRs after I add the missing XML docs to dotnet/runtime

Reply:
Yeah no problem. Let me sign-off.